lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ResamplePercentiles.py

```diff
@@ -487,14 +487,14 @@ def setUp(self):
         data[0] -= 1
         data[1] += 1
         data[2] += 3
-        self.perc_coord = "percentile_over_realization"
+        self.perc_coord = "percentile"
         cube = set_up_cube(data, "air_temperature", "degreesC")
         cube.coord("realization").rename(self.perc_coord)
         cube.coord(self.perc_coord).points = (
             np.array([10, 50, 90]))
         self.percentile_cube = (
             add_forecast_reference_time_and_forecast_period(cube))
-        self.perc_coord_mismatch = "percentile_over_neighbourhood"
+        self.perc_coord_mismatch = "percentile"
```

Comment:
Looking at the context around this test.  The setup function with the same named percentile coordinate now sets up two identical cubes (ie `self.percentile_cube` = `self.percentile_cube_mismatch`).  This and the associated test at line 552 (`test_ok_for_diff_percentile_coord`) are no longer needed and can be removed.

Reply:
Done

Reply:
Thanks.